index.html

```diff
@@ -41,18 +41,21 @@ <h3>Three Favorite Video Games</h3>
                     <li>Super Mario Strikers Charged</li>
                 </ol>
                 <h3>Haiku</h3>
-                    <p>Autumn moonlight-</p>
-                    <p>a worm digs silently</p>
-                    <p>into the chestnut.</p>
-                <cite> by <span class="author">Matsuo Basho</span></cite>
+                <p>
+                    Autumn moonlight-<br>
+                    a worm digs silently<br>
+                    into the chestnut.<br>
+                    <cite> by <span class="author">Matsuo Basho</span></cite>
+                </p>
         </main>
         <footer>
             <hr>
-            <blockquote>"A house divided against itself cannot stand."</blockquote>
-            <cite>— <span class="author">Abraham Lincoln</span></cite>
+            <blockquote>"A house divided against itself cannot stand."<br>
+                <cite>&mdash; <span class="author">Abraham Lincoln</span></cite>
+            </blockquote>
```

Comment:
Instructions were to put `<cite>` inside `<blockquote>`
Instructions were to use an entity character for the dash